package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "snoozetabs",
+  "name": "Snooze Tabs",
```

Comment:
Hmm, that's a page on the defunct SDK that doesn't apply to webextensions. In this case, I think it's fine because npm's package.json rules only say leading & trailing spaces are invalid.